src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -1075,123 +1001,17 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
-            widgetValues = widgetValues.concat(section.widgetValues);
-        });
-
-        // Build the DI Record
-        let diRecord = {};
-
-        for (let [key, value] of Object.entries(fieldData)) {
-            let fieldWrapper = GeFormService.getFieldMappingWrapper(key);
-            if (isNotEmpty(fieldWrapper)) {
-                diRecord[fieldWrapper.Source_Field_API_Name] = value;
-            }
-        }
-
-        // Include any fields from a user selected donation, if
-        // those fields are not already on the diRecord
-        if (dataImportWithDonorData) {
-            for (const [key, value] of Object.entries(dataImportWithDonorData)) {
-                if (!diRecord.hasOwnProperty(key)) {
-                    diRecord[key] = value === null || value.value === null ?
-                        null : value.value || value;
-                }
-            }
-        }
-
-        // resolve widget data to wherever it needs to go, a field on the di object or elsewhere.
-        if (widgetValues) {
-            let diFieldPayloads = [];
-
-            widgetValues.forEach(widgetValue => {
-                if(widgetValue.type === WIDGET_TYPE_DI_FIELD_VALUE && !isUndefined(widgetValue.payload)) {
-                    diFieldPayloads.push(widgetValue.payload);
-                }
-            });
-
-            // diFieldValues is an array of objects where the key is the field name
-            // and the value is the value to be stored
-            const diFieldValuesArray = await Promise.all(diFieldPayloads);
-            let additionalObjectValues = {};
-
-            diFieldValuesArray.forEach(fieldValues => {
-                Object.entries(fieldValues).forEach(([key, value]) => {
-                    if(key === ADDITIONAL_OBJECT_JSON__C) {
-                        // need to accumulate these values so we can serialize them
-                        // before placing them into the data import object
-                        Object.assign(additionalObjectValues, value);
-                    } else {
-                        diRecord[key] = value;
-                    }
-                })
-            });
-            diRecord[ADDITIONAL_OBJECT_JSON__C] = JSON.stringify(additionalObjectValues);
-        }
-        return diRecord;
     }
 
-    /*******************************************************************************
-     * @description Navigates to Gift Entry landing page.
-     */
-    navigateToLandingPage() {
-        const giftEntryTabName =
-            TemplateBuilderService.alignSchemaNSWithEnvironment(
-                GIFT_ENTRY_TAB_NAME,
-                this.namespace
-            );
-        let url = `/lightning/n/${giftEntryTabName}`;
-
-        this[NavigationMixin.Navigate]({
-                type: 'standard__webPage',
-                attributes: {
-                    url: url
-                }
-            },
-            true
-        );
+    handleFormWidgetChange = async (event) => {
```

Comment:
Does this function need to be asynchronous? @dfuller114

Reply:
nah it doesn't. Left over from a previous approach. I'll update that and remove it.

Reply:
👍🏽